recipes-core/image/rpi5-image-minimal-domd.bb

```diff
@@ -5,10 +5,11 @@ require recipes-core/images/core-image-minimal.bb
 # Enable package manager
 EXTRA_IMAGE_FEATURES += "package-management"
 
-RDEPENDS += "rpi-bootfiles"
+DEPENDS:append = " rpi-bootfiles"
```

Comment:
This change has nothing to do with kernel modules. Please move it to a separate commit

Reply:
Deleted